libnavigation-util/src/main/java/com/mapbox/navigation/utils/network/NetworkStatusService.kt

```diff
@@ -61,4 +62,10 @@ class NetworkStatusService(applicationContext: Context) {
             }
         }
     }
+
+    fun cleanup() {
+        if (receiver != null) {
```

Comment:
```suggestion
        receiver?.let {
```

Reply:
I see a lot of discussion about whether using 'let' the way you indicate is appropriate just for passing the variable into a method call rather that acting on the variable directly. I don't feel strongly one way or another except to be consistent with the rest of the team.  I'll make the change you indicate.